diagnostic.md

```diff
@@ -36,19 +36,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+the file path starts with a '/' and there are no .. in the file path
```

Comment:
but is the path absolute or relative? To be clear, '..' is a shorthand that we use to navigate through file structures. However, it is not a part of file paths themselves. The path here is absolute because it starts with the root directory, as signaled by the '/' at the start.